tests/nthash.cpp

```diff
@@ -42,20 +43,6 @@ main()
         TEST_ASSERT_EQ(nthash.hashes()[i],
                        hashes[nthash.get_pos() * nthash.get_hash_num() + i]);
       }
-      if (nthash.get_pos() > 0) {
```

Comment:
Why is this part of code deleted? If it's unnecessary, should we not remove the whole block, starting from `{`?

Reply:
I don't know how this got deleted 🤔  My bad, I'll fix it right away.

Reply:
Fixed in https://github.com/bcgsc/btllib/commit/902d154ccb6060b70609c4daa00605787ffa2916. It happened when I was doing a `git rebase`.